shapedetector.py

```diff
@@ -77,15 +80,24 @@ def detect_shapes():
             c = c.astype(int)
             cv2.drawContours(image, [c], -1, (255, 0, 0), 2)
             cv2.putText(image, shape, (cx, cy), cv2.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 0), 2)
-
-            # cv2.imshow("Image", image)
-            # cv2.waitKey(0)
-
+	
+    #Draw shapes
+    line = cv2.rectangle(blank,(25,25), (190,45), (0,0,255),-1)
+    square = cv2.rectangle(blank, (50,95), (150,200), (0,0,255),-1)
+    circle = cv2.circle(blank, (100,275), 50, (0,0,255),-1)
+    triangle = np.array([(100,350),(50,450),(150,450)])
+    triangle = cv2.drawContours(blank, [triangle], 0, (0,0,255), -1)
+    #Display number of shapes
+    font = cv2.FONT_HERSHEY_SIMPLEX
+    LineNumb = cv2.putText(blank,str(num_shapes['line']),(400, 75),font,3,(0,0,255),2,cv2.LINE_AA)
+    SquareNumb = cv2.putText(blank,str(num_shapes['square']),(400, 175),font,3,(0,0,255),2,cv2.LINE_AA)
+    CircleNumb = cv2.putText(blank,str(num_shapes['circle']),(400, 290),font,3,(0,0,255),2,cv2.LINE_AA)
+    TriangleNumb = cv2.putText(blank,str(num_shapes['triangle']),(400, 430),font,3,(0,0,255),2,cv2.LINE_AA)
     cv2.imshow("Image", image)
-    # cv2.imwrite('image.png', image)
+    #Display Results
+    cv2.imshow("Line", line)
     cv2.waitKey(0)
     return num_shapes
 
-
 num_shapes = detect_shapes()
-print(num_shapes)
+print(num_shapes)
```

Comment:
add a new line at the end of files
```suggestion
print(num_shapes)

```